### DISCUSSION TITLE
About notifications position

### DISCUSSION THREAD

**Discussion by iformas** [valinet/ExplorerPatcher#779] [Q&A]
Thanks for this awesome  tool. I was finally able to move the taskbar to a secondary screen without making it primary.
But sadly notifications keep displaying in primary screen.
There is a way to move them too?

**Comment by terax6669** [comment]
I was about to create the same discussion.

There's no way to fix this on windows right now. Notifications are displaying on your primary screen, which is the screen you are most likely to have a full screen application on. MS has been ignoring this problem for years. I'd be perfect to be able to select the screen for notifications and/or display them on the same screen which has the taskbar.

@valinet is this something EP could do? Or would this be out of scope? I have no idea which part of Windows is responsible for this behavior.

**Comment by terax6669** [comment]
Can we please get an answer one way or another?

**Comment by pyrates999** [comment]
I think this is out of scope for EP.

**Comment by terax6669** [comment]
That's very disappointing.
I have found a soliution... kind of. It doesn't change the notification position, but rather it monitors for new windows appearing and repositions the notification where you specify. It's a little janky, but it seems to be working ok.

https://github.com/RoyRiv3r/notifications-anywhere
